test/performance/latency-controller/README.md

```diff
@@ -1,242 +1,159 @@
 ```
 
-Run Prometheus:
+### Configuration Parameters
+
+| Parameter                   | Description                                                | Required | Default  |
+|-----------------------------|------------------------------------------------------------|----------|----------|
+| `--latency_controller_image`| Container image for the latency controller                 | ✅        | -        |
+| `--binding-name`          | Name of the BindingPolicy to monitor                       | ✅        | -        |
+| `--monitored-namespace`     | Namespace to monitor for workloads                         | ✅        | -        |
+| `--host-context`            | Kubeconfig context for KubeFlex hosting cluster            | ✅        | -        |
+| `--wds-context`             | Kubeconfig context for WDS                                 | ✅        | -        |
+| `--its-context`             | Kubeconfig context for ITS                                 | ✅        | -        |
+| `--wec-files`               | Comma-separated list of WEC configs (name:path)            | ✅        | -        |
+| `--image-pull-policy`       | Image pull policy for the controller                       | ❌        | `Always` |
+| `--controller-verbosity`    | Log verbosity level (0-10)                                 | ❌        | `2`      |
+| `--wds-incluster-file`      | Path to WDS in-cluster config                              | ❌        | -        |
+| `--its-incluster-file`      | Path to ITS in-cluster config                              | ❌        | -        |
+
+Check The latency controller service:
+```
+kubectl get pods -n wds1-system | grep "latency-controller"
+```
 
-```bash
-./prometheus \
-  --config.file=prometheus.yml \
-  --storage.tsdb.path=./data \
-  --web.listen-address=0.0.0.0:9090
+Output:
+```
+NAME                                            READY   STATUS    RESTARTS   AGE
+latency-controller-867f84f4cf-tdl8d             1/1     Running   0          62s
 ```
 
-### Again In Another Terminal (Grafana)
+2. **Access Grafana**:  Import the Grafana dashboard `kubestellar-dashboard.json` into the Grafana UI deployed via KS monitoring setup. 
 
-```bash
-# Download and extract
-wget https://dl.grafana.com/oss/release/grafana-10.4.1.linux-amd64.tar.gz
-tar -xzf grafana-10.4.1.linux-amd64.tar.gz
-cd grafana-10.4.1
-```
+### Step 3: Import KubeStellar Grafana dashboards into the Grafana UI as in Monitoring Tool:
 
-Start Grafana
+After deploying the Latency Controller, import the provided Grafana dashboard to visualize the metrics:
 
-```bash
-./bin/grafana-server web
-```
+1. **Access Grafana**: Connect to your Grafana instance (deployed via KS monitoring setup)  
+2. **Import Dashboard**: Use the provided dashboard JSON configuration  
+3. **Configure Data Source**: Ensure Prometheus is configured as a data source  
+4. **View Metrics**: Monitor latency patterns, identify bottlenecks, and track performance trends 
```

Comment:
We can remove (3) & (4) here. The KS monitoring tools are configures Prometheus as data source by default. Also, you don't need to tell the user to monitor the latency patterns.